mlflow/java/client/src/main/java/org/mlflow/api/proto/Service.java

```diff
@@ -6204,7 +6213,7 @@ private RunInfo(com.google.protobuf.GeneratedMessageV3.Builder<?> builder) {
     }
     private RunInfo() {
       runUuid_ = "";
-      experimentId_ = 0L;
+      experimentId_ = "";
```

Comment:
I think you want this to line up with whatever you decide to make DEFAULT_EXPERIMENT_ID

Reply:
I moved defaults to respective stores. Experiment.DEFAULT_EXPERIMENT_ID -> FileStore.EXP..._ID and SqlAlchemyStore.EXP..._ID.